main.go

```diff
@@ -462,10 +465,26 @@ func main() {
 	td := tdglib.NewToDoGenerator(env.sourceRoot(),
 		includePatterns,
 		excludePatterns,
+		env.authorFromBlame,
 		env.minWords,
 		env.minChars,
 		env.concurrency)
 
+	if env.authorFromBlame {
+		command := "git"
+		args := []string{"config", "--global", "--add", "safe.directory", "/github/workspace"}
+
+		cmd := exec.Command(command, args...)
+
+		cmd.Stdout = os.Stdout
+		cmd.Stderr = os.Stderr
+
+		if err := cmd.Run(); err != nil {
+			fmt.Println("Error running git command.")
```

Comment:
Please output error itself and then stderr and stdout too. e.g. `fmt.Printf("Error running git command: %v\n", err)` and then the same for stdout and stderr